docs/app/interfaces/SiteTheme.ts

```diff
@@ -0,0 +1,11 @@
+export interface SiteTheme {
+    title: string;
+    id: SiteThemeId;
+}
+
+export enum SiteThemeId {
+    Keppel = 'Keppel',
+    MicroFocus = 'MicroFocus',
+    MicroFocus2020 = 'MicroFocus2020',
```

Comment:
Is it necessary to have both SiteTheme and SiteThemeId? I assume the only place you're using it is within the dropdown to display the theme name. If you were to change `MicroFocus2020` to `MicroFocus 2020` as the string value this could then be displayed in the dropdown and would potentially remove the need for both Theme and ThemeId?

Reply:
It's not a good idea to use ID or key values as display text since you would then have to make unnecessary changes to the application in order to modify the display text (and also invalidate any prior value for the setting).